crest/Assets/Crest/Crest-Examples/Shared/Scripts/UnderwaterEnvironmentalLighting.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Crest Ocean System
+
+// This file is subject to the MIT License as seen in the root of this folder structure (LICENSE)
+
+using UnityEngine;
+
+namespace Crest
+{
+    /// <summary>
+    /// Provides outer scattering based on the camera's underwater depth. It scales down environmental lighting
```

Comment:
Minor typo - outer scattering should be 'outscattering'

Reply:
Thanks for the correction. For some reason it has been outer scattering in my mind this whole time lol